assets/js/admin/admin.js

```diff
@@ -221,4 +221,61 @@
 		$(this).append(video);
 
 	});
+
+	// Toggles edit state.
+	function toggleEditTitle( e ) {
+		e.preventDefault();
+
+		var $ele = $( '#edit-form-name' ),
+			$input_title = $ele.siblings( '#evf-edit-form-name' );
+		// Toggle disabled property.
+		$input_title.prop( 'disabled' , function(_, val) {
+			return ! val;
+		});
+		if ( ! $input_title.hasClass( 'everst-forms-name-editing' ) ) {
+			$input_title.focus();
+		}
+		$input_title.toggleClass( 'everst-forms-name-editing' );
+	}
+
+	// Delegates event to toggleEditTitle() on clicking
+	$( '#edit-form-name' ).on( 'click', function( e ) {
+		e.stopPropagation();
+		if ( '' !== $( '#evf-edit-form-name').val().trim() ) {
+			toggleEditTitle( e );
+		}
+	});
+
+	// Apply the title change to form name field.
+	$( '#evf-edit-form-name' ).on( 'change', function( e ) {
```

Comment:
Below `on` events are in the new line so for this line why it is missing?

Reply:
The on statement indentations are now conforming to the coding style.